authentication/views.py

```diff
@@ -1,13 +1,47 @@
-from django.shortcuts import render
-from django.contrib.auth import login,logout,authenticate
-# Create your views here.
-
+from django.http import *
+from django.shortcuts import redirect,render
+from django.template import RequestContext
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth import authenticate, login, logout
+from .forms import *
 
 def loginView(request):
-    pass
+    if request.user.is_authenticated:
+        return redirect('/')
+    username = password = ''
+    context = { 'message' : "none"}
+    template_name = 'store/login.html'
+    if request.POST:
+        username = request.POST['username']
+        password = request.POST['password']
```

Comment:
You are directly accessing POST data without checking if it even exists. This may lead to server crash if a user access this endpoint with invalid request data. The good behavior would have been to throw a client error (400), rather than server error (500).